runtime/ddr/run_omrddrgen.mk.ftl

```diff
@@ -114,8 +114,12 @@ clean :
 
 $(DDR_BLOB) : $(TOP_DIR)/ddrgen$(UMA_DOT_EXE) $(DDR_MACRO_LIST) excludes $(wildcard overrides*)
 	@echo "Running ddrgen to generate $(notdir $@) and $(notdir $(DDR_SUPERSET_FILE))"
+<#if uma.spec.type.linux && uma.spec.processor.aarch64 && uma.spec.flags.env_crossbuild.enabled>
+	@qemu-aarch64 $(TOP_DIR)/ddrgen $(DDR_OPTIONS) $(DDR_INPUT_FILES)
+<#else>
```

Comment:
I cross-compiled for aarch64 on my x86_64 Ubuntu system (without ddr). With `QEMU_LD_PREFIX` set appropriately, I can run `qemu-aarch64 java` or directly (without using `qemu-aarch64`): the results are the same (unfortunately it crashes with illegal instruction). Should we ensure `QEMU_LD_PREFIX` is set in the build rather than make changes such as this?

Reply:
I searched for information on `QEMU_LD_PREFIX`, but I don't understand how it works.

Reply:
There is no directory like /usr/aarch64-linux-gnu to be set to `QEMU_LD_PREFIX` in my old Docker environment for cross-building OpenJ9 AArch64 Linux.  It seems I need to update my environment.

Reply:
In the container I have, the environment variable would be set via:
```
export QEMU_LD_PREFIX=/root/gcc-linaro-7.5.0-2019.12-x86_64_aarch64-linux-gnu/aarch64-linux-gnu/libc
```

Reply:
Thank you.
I made sure I was able to run ddrgen in my cross-building environment by setting `QEMU_LD_PREFIX`.
I don't need my change in runtime/ddr/run_omrddrgen.mk.ftl to run ddrgen.

Perhaps I would add comments to build instructions for `QEMU_LD_PREFIX`.